components/member-profile/active-task/active-task.module.scss

```diff
@@ -39,4 +54,76 @@
   .taskSection .featureTitle{
     font-size: 1rem;
   }
+}
+
+.progressSection{
+  width: 35%;
+  padding: 0.1em 2em;
+  display: flex;
+  flex-direction: column;
+  align-items: space-between;
+}
+
+.progressText{
+  width: 162px;
+height: 31px;
+  font-family: Helvetica Neue;
+font-style: normal;
+font-weight: normal;
+font-size: 20px;
+line-height: 24px;
+display: flex;
+justify-content: center;
+color: #ABA9A9;
+}
+
+.progressBar {
+  width: 100%;
+  height: 5px;
+  position: relative;
+  background-color:#e5e5e5;
+  border: 1px solid rgba(0, 0, 0, 0.24);
+}
+
+
+
+.showProgressOrange{
+  width: 1%;
+  height: 3px;
+  background-color:orange
+}
+
+.showProgressRed{
+  width: 1%;
+  height: 3px;
+  background-color: red
+}
+
+.showProgressYellow{
+  width: 1%;
+  height: 3px;
+  background-color: yellow
+}
+
+.showProgressGreen {
+  width: 1%;
+  height: 3px;
+  background-color: green;
+}
+
+.showPercentCompletedText{
+  display: flex;
+  justify-content: center;
+  padding: 0.25em 0;
+height: 25px;
+width: 164px;
+color: #BEBEBE;
+font-family: Helvetica Neue;
+font-style: normal;
+font-weight: normal;
+font-size: 12px;
```

Comment:
Is it better to have a single line for font property?
https://css-tricks.com/almanac/properties/f/font/